e2e/cypress/integration/system_console/ui_and_api/customization_spec.js

```diff
@@ -0,0 +1,42 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+// ***************************************************************
+// - [#] indicates a test step (e.g. # Go to a page)
+// - [*] indicates an assertion (e.g. * Check the title)
+// - Use element ID when selecting an element. Create one if none.
+// ***************************************************************
+
+describe('Customization', () => {
+    before(() => {
+        // # Login as sysadmin and visit customization system console page
+        cy.apiLogin('sysadmin');
+        cy.visit('/admin_console/site_config/customization');
+    });
+
+    it('SC20336 - Can change Custom Brand Image setting', () => {
+        // # Set Enable Custom Branding to true to be able to upload custom brand image
+        cy.get('[id="TeamSettings.EnableCustomBrandtrue"]').check();
```

Comment:
I'm guessing the `true` suffix distinguishes it from the `false` radio option?

@saturninoabril, can you elaborate on how this becomes scalable? It seems like /everything/ in the app must have an id in order to be tested, instead of doing something more "human" like and searching for an option by name and manipulating that. Deferring to e2e best practices here of course.

Reply:
I understand and this may change soon.
I believe such approach is per [Cypress best practices](https://docs.cypress.io/guides/references/best-practices.html#Selecting-Elements) but I'm also starting to realize some of the better approached being offered by [testing library](https://testing-library.com/docs/dom-testing-library/api-queries), similar to being more "human".  I'll be gathering info, feedback like this and observation, and hopefully come up with a proposal of best practices in writing end-to-end tests (subject to general approval).
For the meantime, let's have it as is until we have general agreement on how to do things.